sdks/python/apache_beam/runners/dataflow/dataflow_runner_test.py

```diff
@@ -102,7 +102,8 @@ def setUp(self):
         '--staging_location=ignored',
         '--temp_location=/dev/null',
         '--no_auth',
-        '--dry_run=True'
+        '--dry_run=True',
+        '--sdk_location=container'
```

Comment:
Why this change?

Reply:
The test tries to download a dev version of apache-beam dependency (which indeed does not exist in pypi) when it constructs the environment.